packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/administration/telecom-telephony-billing-account-administration.controller.js

```diff
@@ -2,7 +2,11 @@ angular
   .module('managerApp')
   .controller(
     'TelecomTelephonyBillingAccountAdministrationCtrl',
-    function TelecomTelephonyBillingAccountAdministrationCtrl($translate) {
+    function TelecomTelephonyBillingAccountAdministrationCtrl(
+      $translate,
+      $stateParams,
```

Comment:
Could you please order alphabetically ? :)

Reply:
done